oc-chef-pedant/spec/api/oc_id_spec.rb

```diff
@@ -2,6 +2,38 @@
 require 'openssl'
 
```

Comment:
I added a new `untested_endpoint` method that we can use to document issues like this.